standards/custom-applications.md

```diff
@@ -0,0 +1,92 @@
+---
+category: Security Log Collection
+expires: 2020-03-01
+---
+
+# Custom Applications
+
+We have developed a series of logging requirements for custom applications, such as digital services, applications materially customised that they can reasonably be considered bespoke/custom for the MOJ and line of business applications at different maturity tiers in order to support defenisve cyber security, such as detecting breaches.
+
+## Baseline Maturity Tier
+
+### 1. User directory services
+Log Collection Principle(s): 1, 2
```

Comment:
A thought for the future maybe - cross linking to the principles?

Reply:
Yes, relative links throughout would be good in the future.